File: src/main/java/xsbti/InteractiveConsoleResponse.java
/*
 * Zinc - The incremental compiler for Scala.
 * Copyright 2011 - 2017, Lightbend, Inc.
 * Copyright 2008 - 2010, Mark Harrah
 * This software is released under the terms written in LICENSE.
 */

package xsbti;

/** Public interface for repl responses. */
public interface InteractiveConsoleResponse {
  InteractiveConsoleResult result();

  String output();
}

File: src/compiler/scala/tools/nsc/incremental/InteractiveConsoleResponse.scala
/*
 * Zinc - The incremental compiler for Scala.
 * Copyright Lightbend, Inc. and Mark Harrah
 *
 * Licensed under Apache License 2.0
 * (http://www.apache.org/licenses/LICENSE-2.0).
 *
 * See the NOTICE file distributed with this work for
 * additional information regarding copyright ownership.
 */

package xsbt

import xsbti.InteractiveConsoleResult

case class InteractiveConsoleResponse(result: InteractiveConsoleResult, output: String)
    extends xsbti.InteractiveConsoleResponse
